content/css/jenkins.css

```diff
@@ -1449,3 +1450,4 @@ p.details {
 .supporters a {
   font-weight: bold;
 }
+
```

Comment:
@batmat Got the point, will take care from now. However I have a suggestion, if we could add a pre-receive hook at server end so that it can check for the code formatting and indentation.